docs/fides/docs/quickstart/docker.md

```diff
@@ -66,7 +36,7 @@ services:
     environment:
       - FIDESCTL__CLI__SERVER_HOST=http://fidesctl
       - FIDESCTL__CLI__SERVER_PORT=8080
-      - FIDESCTL__API__DATABASE_URL=postgres:fidesctl@fidesctl-db:5432/fidesctl
+      - FIDESCTL__API__DATABASE_URL=postgresql://postgres:fidesctl@fidesctl-db:5432/fidesctl
```

Comment:
@conceptualshark this looks good to me, but I also want to make sure someone that knows docker-compose better than I do gives this a look as well. I'm seeing in main it's actually `FIDESCTL__API__DATABASE_HOST: "fidesctl-db"`, but maybe okay bc it's defaulting to the local db spun up with the docker image, might be different when we can't control that.

Reply:
Yea @iamkelllly / @conceptualshark - this should be aligned with [the current `docker-compose.yml` on `main`](https://github.com/ethyca/fides/blob/62372660f57c181651d1ba6a2584ccdd5ac1e1f6/docker-compose.yml#L26). The `FIDESCTL__API__DATABASE_URL` value is technically a sensitive value, since it contains access credentials and enables anyone who knows the value to connect directly to the DB. In the most current iteration of our config workflow, the `FIDESCTL__API__DATABASE_URL` value is automatically constructed from other config values (including `FIDESCTL__API__DATABASE_HOST`), and not expected to be populated directly.